test/deprecated/classes/owned-clear.chpl

```diff
@@ -0,0 +1,8 @@
+class A {
```

Comment:
I suggest a comment at the top including your name and "deprecated for 1.30". So we know when this feature can be removed altogether. Adjust the line number in .good, then.